internal/compression/types.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compression
+
+var (
+	supportedCompressionTypes = map[string]bool{
```

Comment:
Perhaps we could provide more context as to what compression is for.

How about explicitly saying `supportedGRPCCompressionTypes`
so
```go
// Add more supported gRPC compression types here.
var supportedGRPCCompressionTypes = map[string]bool {
    "gzip": true,
}
```
and then

```go
func SupportedGRPCCompression(compressionType string) bool {
     supported, ok := supportedCompressionTypes[compressionType]
    return ok && supported
}
```

Reply:
Please see my comment about not making this a package due to ambiguity.